packages/app/src/components/search/SearchModal.tsx

```diff
@@ -206,62 +192,51 @@ export const SearchModal = ({ toggleModal }: { toggleModal: () => void }) => {
               {({ results }) => (
                 <List>
                   {results.map(({ type, document, highlight, rank }) => {
-                    let resultItem;
                     switch (type) {
                       case 'software-catalog':
-                        resultItem = (
```

Comment:
This will conflict with https://github.com/backstage/backstage/pull/15855

Reply:
Will wait until https://github.com/backstage/backstage/pull/15855 is merged in 💯